page-objects/pages/coursePages/CRSSAddCoursePage.js

```diff
@@ -1 +1,224 @@
+};
+
+//Participants section
+const classContainer = '[data-testid="class_container"]';
+const studentsContainer = '[data-testid="students_container"]';
+
+//Final section
+const createNewCourseBtn = '[data-testid="einen-weiteren-kurs-anlegen-btn"]';
+const goToCourseListBtn = '[data-testid="zur-uebersicht-btn"]';
+
+module.exports = {
+	goToNextSection: async function () {
+		await wh.waitAndClick(nextSectionBtn);
+	},
+
+	getListOfSelected: async function (containerSelector) {
+		const container = await driver.$(containerSelector);
+		const listOfElements = await container.$$(chosenInput);
+		return await eh.getTextListFromListOfElements(listOfElements);
+	},
+
+	isDefaultValueInContainer: async function (containerSelector, defaultText) {
+		const container = await driver.$(containerSelector);
+		const listOfElements = await container.$$(chosenDefInput);
+		const valueList = await eh.getValueListFromListOfElements(listOfElements);
+		const isOnlyOneText = valueList.length == 1;
+		await expect(isOnlyOneText).is.equal(true);
+		await expect(valueList).includes(defaultText);
+	},
+
+	sectionIsDisplayed: async function (sectionNumber) {
+		const sectionToCheck =
+			sectionNumber == 1 ? sectionNumber : sectionNumber - 1;
+		const selector = this.getSectionSelector(sectionToCheck);
+		const element = await driver.$(selector);
+		const hasChildren = (await element.$$(".//*")).length > 0;
+
+		if (sectionNumber == 1) {
+			await expect(await eh.isElementPresent(selector)).to.equal(true);
+			await expect(hasChildren).to.equal(false);
+		} else {
+			await expect(hasChildren).to.equal(true);
+		}
+	},
+
+	sectionIsNotDisplayed: async function (sectionNumber) {
+		const sectionToCheck =
+			sectionNumber == 1 ? sectionNumber : sectionNumber - 1;
+		const element = await driver.$(this.getSectionSelector(sectionToCheck));
+		const hasChildren = (await element.$$(".//*").length) > 0;
+
+		if (sectionNumber == 1) {
+			await expect(await eh.isElementPresent(sectionToCheck)).to.equal(
+				false
+			);
+		} else {
+			await expect(hasChildren).to.equal(false);
+		}
+	},
+
+	getSectionSelector: function (sectionNumber) {
+		let selector;
+		switch (sectionNumber) {
+			case 1:
+				selector = section.one;
+				break;
+			case 2:
+				selector = section.two;
+				break;
+			case 3:
+				selector = section.three;
+				break;
+			default:
+				console.error(
+					`This section: ${sectionNumber} does not exist on the list of possible choices`
+				);
+				break;
+		}
+		return selector;
+	},
+
+	getUserName: async function () {
+		const cookie = await driver.getCookies(["jwt"]);
+		const jwt = cookie[0].value;
+		const info = await axios.request({
+			url: "http://localhost:3030/me",
+			method: "get",
+			headers: {
+				Authorization: `${jwt}`,
+			},
+		});
+		const firstName = info.data.firstName;
+		const lastName = info.data.lastName;
+		return firstName + " " + lastName;
+	},
+
+	//Course data section
+	courseNameIsNotEntered: async function () {
```

Comment:
the name of methods leads me to the assumption that you are checking that the input field is empty, but you are checking  if the placeholder test is set as expected
- please also add the check that the input field is empty